### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by Nora-Olivia-Ammann**

### CODE REVIEW

File: index.markdown
Comment (Nora-Olivia-Ammann) [inline index.markdown:1]: Currently the images use the old prefix `trip-onto`. That should be updated to `js`. The graphml files are not here correct? @SepidehAlassi

File: journeyStar.ttl [hunk 1]
```diff
@@ -3,39 +3,30 @@
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix currencyA: <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/> .
-@prefix currency: <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/>.
```
Comment (Nora-Olivia-Ammann) [inline journeyStar.ttl:6]: These prefixes are not used

File: journeyStar.ttl [hunk 2]
```diff
@@ -3,39 +3,30 @@
...
 ### http://journey-star.dhlab.unibas.ch/ontology/JourneyStar#Person ###
 js:Person rdf:type owl:Class ;
     rdfs:subClassOf schema:Person;
     rdfs:label "Person"@de ,
-                "Person"@en ;
+               "Person"@en ;
     rdfs:comment "A person."@en .
 
-### http://journey-star.dhlab.unibas.ch/ontology/JourneyStar#Event ###
-js:Event rdf:type owl:Class ;
```
Comment (Nora-Olivia-Ammann) [inline journeyStar.ttl:34]: I grouped the classes and properties based on sub-classing and sub-properties.

File: journeyStar_shacl.ttl [hunk 3]
```diff
@@ -203,33 +201,33 @@ js-shacl:JourneyShape a sh:NodeShape;
     ] ,
     [
         sh:path js:origin;
+        sh:minCount 1;
```
Comment (Nora-Olivia-Ammann) [inline journeyStar_shacl.ttl:204]: Consistent ordering of types of properties

File: journeyStar.ttl [hunk 4]
```diff
@@ -302,24 +320,28 @@ js:hasCost rdf:type owl:DatatypeProperty ;
                    "has cost"@en ;
         rdfs:comment "The cost of an activity, transport, or accomodation."@en .
 
+
```
Comment (Nora-Olivia-Ammann) [inline journeyStar.ttl:323]: consistent spacing

File: journeyStar_shacl.ttl [hunk 5]
```diff
@@ -2,11 +2,8 @@
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix currencyA: <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/CurrencyAmount/> .
-@prefix currency: <https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/>.
 @prefix schema: <https://schema.org/> .
 @prefix dbr: <http://dbpedia.org/resource/> .
-@prefix dbo: <http://dbpedia.org/ontology/> .
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix js-shacl: <http://journey-star.dhlab.unibas.ch/shacl/trip-shacl#> .
```
Comment (Nora-Olivia-Ammann) [inline journeyStar_shacl.ttl]: I would suggest to change the namespace to `journey-shacl` for consistency's sake

File: journeyStar.ttl [hunk 6]
```diff
@@ -45,103 +36,115 @@ js:Location rdf:type owl:Class ;
...
 ### http://journey-star.dhlab.unibas.ch/ontology/JourneyStar#Activity ###
 js:Activity rdf:type owl:Class ;
         rdfs:subClassOf activity:Activity , js:Event ;
         rdfs:label "Aktivität"@de,
-                    "Activity"@en ;
+                   "Activity"@en ;
         rdfs:comment "An activity involves movement and has duration."@en .
 
 
+# http://journey-star.dhlab.unibas.ch/ontology/JourneyStar#SightSeeing #
```
Comment (SepidehAlassi) [inline journeyStar.ttl]: remove the ` #` from the end of all URLs put in comments

**Review by SepidehAlassi — CHANGES_REQUESTED** [review]
Please remove the `#` from end of class and property URLs, they can cause confusion. Then it is good to go

File: index.markdown
Comment (SepidehAlassi) [reply index.markdown:1]: yes, graphml file are old. They should be removed from the repo, and the images in docs should be updated.

File: journeyStar_shacl.ttl (hunk 5 above)
Comment (SepidehAlassi) [reply journeyStar_shacl.ttl]: yes, it should be corrected.

File: index.markdown
Comment (Nora-Olivia-Ammann) [reply index.markdown:1]: Do you still have the files? Because I think it would be fasted to fix the old ones.

**Pull request merged**